docs/Other SDL Features/Uploading Images/index.md

```diff
@@ -105,7 +105,7 @@ sdlManager.fileManager.upload(artwork: artwork) { (success, artworkName, bytesAv
 
 @![android,javaSE,javaEE]
 ```java
-SdlArtwork artwork = new SdlArtwork("image_name", FileType.GRAPHIC_PNG, <image byte[]>, false);
+SdlArtwork artwork = new SdlArtwork("image_name", FileType.GRAPHIC_PNG, <#image#>, false);
```

Comment:
Should this be made more clear which constructor is used or what data type actually goes here? such as the following
<#image_resourceId#>
<#image_uri#>
<#image_byteArray#>